converter_schwab.py

```diff
@@ -0,0 +1,169 @@
+                schwab_espp_events.append(ESPPRow.from_schwab_json(e).to_dict())
+
+            # assumption behind RSU: each grant has its own vest/deposit event
+            # assumption behind RSU: award-id, year, and month are unique to each
+            # deposit/lapse event (day of deposit and lapse might differ)
+            elif (
+                e["Action"] == "Lapse" and e["Description"] == "Restricted Stock Lapse"
+            ):
+                tmp, award_id = RSURow.from_schwab_lapse_json(e)
+                key = (tmp.date.year, tmp.date.month, award_id)
+                if key in schwab_rsu_lapse_events:
+                    raise RuntimeError("Found duplicated RSU Lapse event: {tmp}")
+                schwab_rsu_lapse_events[key] = tmp
+
+            elif e["Action"] == "Deposit" and e["Description"] == "RS":
+                tmp, award_id = RSURow.from_schwab_deposit_json(e)
+                key = (tmp.date.year, tmp.date.month, award_id)
+                if key in schwab_rsu_deposit_events:
+                    raise RuntimeError(f"Found duplicated RSU deposit event: {tmp}")
+                schwab_rsu_deposit_events[key] = tmp
+
+            elif e["Action"] == "Dividend" and e["Description"] == "Credit":
+                tmp = DividendRow.from_schwab_json(e)
+                if len(schwab_dividend_events) > 0 and isinstance(
+                    schwab_dividend_events[-1], TaxWithholdingRow
+                ):
+                    tax = schwab_dividend_events.pop(-1)
+                    tmp.tax_withholding = tax.amount
+                    schwab_dividend_events.append(tmp.to_dict())
+                else:
+                    schwab_dividend_events.append(tmp)
+
+            elif e["Action"] == "Sale" and e["Description"] == "Share Sale":
+                schwab_sell_events.append(SellOrderRow.from_schwab_json(e).to_dict())
+
+            elif (
+                e["Action"] == "Wire Transfer"
+                and e["Description"] == "Cash Disbursement"
+            ):
+                schwab_wire_events.append(
+                    CurrencyConversionRow.from_schwab_json(e).to_dict()
+                )
+
+            elif e["Action"] == "Tax Withholding" and e["Description"] == "Debit":
+                tmp = TaxWithholdingRow.from_schwab_json(e)
+                if len(schwab_dividend_events) > 0 and isinstance(
+                    schwab_dividend_events[-1], DividendRow
+                ):
+                    div = schwab_dividend_events.pop(-1)
+                    div.tax_withholding = tmp.amount
+                    schwab_dividend_events.append(div.to_dict())
+                else:
+                    schwab_dividend_events.append(tmp)
+
+            else:
+                act = e["Action"]
+                des = e["Description"]
+                sym = e["Symbol"]
+                date = e["Date"]
+                warnings.warn(f"skipping {act} on {date} ({sym}: {des})")
```

Comment:
I get this warning message: `skipping Transfer on 01/11/2022 (XXXX: Share Transfer)`. What is the user  expected to do with this warning?

If it's not harmful and actionable, I'd remove this warning entirely.